external/SConscript-libgmp

```diff
@@ -24,13 +24,18 @@ elif not os.path.exists("gmp-6.1.2/configure"):
         open("gmp-6.1.2/gmp.h", "w").write(gmp_h)
         config_h = open("gmp-6.1.2/config.in").read()
         config_h = config_h.replace("#undef WANT_TMP_ALLOCA", "#define WANT_TMP_ALLOCA 1")
+        config_h = config_h.replace("#undef inline", "/*#undef inline*/")
         open("gmp-6.1.2/config.h", "w").write(config_h)
 
 if sys.platform == "win32":
     if not env["RELEASE"]:
-        gmpenv.Append(CFLAGS=["/MTd"])
+        gmpenv.Append(CFLAGS=["/MTd",
```

Comment:
If you're doing the config.h change, then you don't need this stuff at all.

Reply:
Fixed